api/types/oidc.go

```diff
@@ -62,8 +61,6 @@ type OIDCConnector interface {
 	SetClientID(string)
 	// SetIssuerURL sets the endpoint of the provider
 	SetIssuerURL(string)
-	// SetRedirectURL sets RedirectURL
-	SetRedirectURL(string)
```

Comment:
Don't we need `SetRedirectURLs(string)`?

Reply:
I don't think so, if it's needed in the future it can be added at that time.